src/masonite/hashing/drivers/BcryptHasher.py

```diff
@@ -0,0 +1,25 @@
+import bcrypt
+
+
+class BcryptHasher:
+    def __init__(self, options={}):
+        self.options = options
+
+    def set_options(self, options):
+        self.options = options
+        return self
+
+    def make(self, string):
+        rounds = self.options.get("rounds")
+        salt = bcrypt.gensalt(rounds=rounds)
+        return str(bcrypt.hashpw(bytes(string, "utf-8"), salt))
+
+    def check(self, plain_string, hashed_string):
+        rounds = self.options.get("rounds")
+        # TODO: here we should take care of comparing use same rounds count
+        # byt bcrypt does not expose public api to do this !
+        # salt = bcrypt.gensalt(rounds=rounds)
+        # ref_hashed = bcrypt.hashpw(bytes(plain_string, "utf-8"), salt)
```

Comment:
I am pretty sure with bcrypt, you do not need to specify the salt when checking the password. Just check the byte version of the supplied password with the hashed version of it

Reply:
The salt is already included in the password hash that we store in the database

Reply:
That's right.